pkg/asset/installconfig/powervs/session.go

```diff
@@ -23,6 +25,7 @@ var (
 type Session struct {
 	Session *ibmpisession.IBMPISession
 	Creds   *UserCredentials
+	Zone    *icibmcloud.Zone
```

Comment:
I think this Session object is turning into a catchall for extra variables. The Session was the PI Session before, which doesn't have anything to do with a DNS zone. I think we need to organize all this a bit more clearly. For now, though, we can leave it. But can you just rename Zone to DNSZone? There are a two different types zones in this part of the code, and the fact that the client only named the DNS zone `Zone` is a bit confusing in this context.